### PR TITLE
1984 Update the registration page texts

### PR DESCRIPTION
**Pull request by ordabach**
# What does this PR do?

## PR Checklist
* [X] Have you added an explanation of what your changes do and why you'd like to include them?
* [] Is the TravisCI build passing?
* []Was the CHANGELOG.md updated to reflect the changes?
* ~Was the documentation framework updated to reflect the changes?
* ~Have you checked that you haven't introduced any duplicate code?~

## Testing Checklist

* ~Added relevant unit tests?~
* ~Do all unit tests pass?~
* ~Do all end-to-end tests pass?~
* ~Any other testing performed?~
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [X] If applicable, add screenshots or log transcripts of the feature working

Screenshot 2023-07-06 at 13 43 50

### CODE REVIEW

File: monkey/monkey_island/cc/ui/src/components/pages/RegisterPage.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ class RegisterPageComponent extends React.Component {
             <Row>
               <Col lg={8} md={8} sm={8}>
                 <h1 className='reg-title'>First time?</h1>
-                <h3 className='reg-subtitle'>Let's secure your Monkey Island!</h3>
+                <h3 className='reg-subtitle'>Create a username and password to secure the island</h3>
```
Comment (shreyamalviya) [inline monkey/monkey_island/cc/ui/src/components/pages/RegisterPage.js:68]:
```suggestion
                <h3 className='reg-subtitle'>Create a username and password to secure the Island</h3>
```

File: monkey/monkey_island/cc/ui/src/components/pages/RegisterPage.js [hunk 2]
```diff
@@ -75,7 +75,7 @@ class RegisterPageComponent extends React.Component {
                         this.state.loading ?
                           <LoadingIcon/>
                           :
-                          'Let\'s go!'
+                          'Sign Up!'
```
Comment (shreyamalviya) [inline monkey/monkey_island/cc/ui/src/components/pages/RegisterPage.js:78]:
```suggestion
                          'Sign Up'
```

File: monkey/monkey_island/cc/ui/src/components/pages/RegisterPage.js [hunk 3]
```diff
@@ -65,7 +65,7 @@ class RegisterPageComponent extends React.Component {
             <Row>
               <Col lg={8} md={8} sm={8}>
                 <h1 className='reg-title'>First time?</h1>
```
Comment (shreyamalviya) [inline monkey/monkey_island/cc/ui/src/components/pages/RegisterPage.js:67]: Maybe we should change this too. WDYT?

**Review by shreyamalviya — CHANGES_REQUESTED** [review]

File: monkey/monkey_island/cc/ui/src/components/pages/RegisterPage.js (hunk 3 above)
Comment (ilija-lazoroski) [reply monkey/monkey_island/cc/ui/src/components/pages/RegisterPage.js:67]: +1

Comment (shreyamalviya) [reply monkey/monkey_island/cc/ui/src/components/pages/RegisterPage.js:67]: Or change the title to "Sign Up", and leave the button's text as "Let's go!"

**Review by mssalvatore — APPROVED** [review]

**Comment by shreyamalviya** [comment]
Updated:

image

**Pull request merged**